sql/api/src/main/scala/org/apache/spark/sql/util/ArrowUtils.scala

```diff
@@ -51,7 +51,7 @@ private[sql] object ArrowUtils {
     case BinaryType if !largeVarTypes => ArrowType.Binary.INSTANCE
     case _: StringType if largeVarTypes => ArrowType.LargeUtf8.INSTANCE
     case BinaryType if largeVarTypes => ArrowType.LargeBinary.INSTANCE
-    case DecimalType.Fixed(precision, scale) => new ArrowType.Decimal(precision, scale)
+    case DecimalType.Fixed(precision, scale) => new ArrowType.Decimal(precision, scale, 8 * 16)
```

Comment:
Can we use a named `val` to replace this magic number?

Reply:
I refer to the style of lines 44-47

Reply:
OK